public/index.html

```diff
@@ -24,7 +24,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link
       rel="stylesheet"
-      href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700&family=Roboto:300,400,500,700&family=Roboto+Slab:700&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&family=Roboto:wght@300;400;500;700&family=Roboto+Slab:wght@700&display=swap"
```

Comment:
The font still is not importing for me.
![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/62856626/6cc408c5-540c-459e-96d2-8116591bbcdb)
![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/62856626/08687916-2d34-4492-9dd2-3283d629e1c0)